dev/drivers/scripts/cam/stats/jevs_cam_nam_firewxnest_grid2obs_stats.sh

```diff
@@ -53,6 +38,8 @@ export MODELNAME=nam_firewxnest
 export modsys=nam
 export mod_ver=${nam_ver}
 
+source $HOMEevs/modulefiles/$COMPONENT/${COMPONENT}_${STEP}.sh
+
 export MET_bin_exec=bin
```

Comment:
@PerryShafran-NOAA let's remove line 43!

Reply:
This is done.

Perry